### PR TITLE
Add the auto-created .mono/ folder to .gitignore

### PR DESCRIPTION
**Pull request by carlossanlop**
I noticed we don't have a `.mono/` folder commited at the root of the repo, but it gets created sometimes when working with native code. I'm adding the entry to .gitignore ensures we don't accidentally get it included in a PR in the future.

### CODE REVIEW

**Review by jkoritzinsky — APPROVED** [review]

**Comment by carlossanlop** [comment]
Looks like the CI is breaking, but I'm not yet sure if it's due to my change.

**Comment by carlossanlop** [comment]
I don't have a consistent repro to get the .mono/ folder created at the top level. Since this seems suspiciously related to the CI break, I'll close the PR and if I can get a consistent repro, I'll investigate why the folder is showing up.

**Comment by jkoritzinsky** [comment]
I think the CI break is unrelated to this PR. I think it's related to my docker image changes (revert PR already open for that)

**Comment by carlossanlop** [comment]
Ah, okay, thanks for checking! I'll reopen after you merge your PR.

**Comment by jkoritzinsky** [comment]
Can you approve the revert if you have a chance?

https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1038

**Comment by carlossanlop** [comment]
Done!

**Pull request closed without merging**
